pipeline/src/node_properties/track_properties.rs

```diff
@@ -3,6 +3,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct TrackProperties {
     pub tracker: Tracker,
+    pub profile: Option<TrackerProfile>,
```

Comment:
```suggestion
    #[serde(default)]
    pub profile: TrackerProfile,
```

Reply:
More info on serde attributes can be found here if you're curious: https://serde.rs/field-attrs.html